### PR TITLE
Feature: Footer Redesign

### PR DESCRIPTION
**Pull request by KevinMulhern**
Because:
* We want to move all our CSS to Tailwind and Ada has designed us a new Footer.

This commit:
* Implements Ada's new design in Tailwind.
* The mobile design is limited at the moment, we ran into some issues with Bootstrap overriding tailwind classes and will revisit this when Bootstrap is removed.
* Use Fontawesome through its JS package instead of through a CDN.

#### Checklist
 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide)?
 - [x] You have verified the tests and linters all pass against your changes?
 - [x] You have included automated tests for your changes where applicable?

### CODE REVIEW

File: app/assets/stylesheets/components/courses/course_section.scss [hunk 1]
```diff
@@ -18,6 +18,6 @@
   }
 
   &:last-child {
-    margin-bottom: 50px;
+    margin-bottom: 100px;
```
Comment (leosoaivan) [inline app/assets/stylesheets/components/courses/course_section.scss:21]: 👀

File: app/views/shared/_footer_cta.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<% if user_signed_in? %>
+  <div class="border border-gray-300 rounded-md p-8 hidden lg:block">
+    <h3 class="text-3xl font-bold text-gray-700">Have a question?</h2>
+    <p class="py-3 text-gray-800 text-md font-light w-96">Chat with the Odin community on discord!</p>
```
Comment (leosoaivan) [inline app/views/shared/_footer_cta.html.erb:4]: Capitalize `discord`

**Review by leosoaivan — APPROVED** [review]
Approving, minor grammar nitpick

File: app/views/shared/_footer_cta.html.erb (hunk 2 above)
Comment (KevinMulhern) [reply app/views/shared/_footer_cta.html.erb:4]: Nice spot 💪

**Pull request merged**